examples/air/main.py

```diff
@@ -246,6 +246,7 @@ def per_param_optim_args(module_name, param_name):
 
 
 if __name__ == '__main__':
+    assert pyro.__version__.startswith('0.3.0')
```

Comment:
'0.3' probably works since patches shouldnt break examples

Reply:
While `starswith('0.3')` would work while `pyro.__version__ == 0.3.0`, we'll eventually need `startswith('0.3.1')` when we bump to `pyro.__version__ == 0.3.1`. It seems simpler to pin the whole version.